packages/@uppy/companion/src/server/socket.js

```diff
@@ -48,6 +48,16 @@ module.exports = (server) => {
     emitter().emit(`connection:${token}`)
     emitter().on(token, send)
 
+    ws.on('error', (err) => {
+      // https://github.com/websockets/ws/issues/1543
+      // https://github.com/websockets/ws/blob/b73b11828d166e9692a9bffe9c01a7e93bab04a8/test/receiver.test.js#L936
+      if (err?.name === 'RangeError' && 'code' in err && err.code === 'WS_ERR_UNSUPPORTED_MESSAGE_LENGTH') {
```

Comment:
Do we need to check for `'code' in err`?
```suggestion
      if (err?.name === 'RangeError' && err.code === 'WS_ERR_UNSUPPORTED_MESSAGE_LENGTH') {
```

Reply:
yes or else we get a ts error